<?xml version="1.0" encoding="UTF-8"?>
<<<<<<< HEAD
<testsuite name="com.linecorp.bot.client.ExceptionConverterTest" tests="3" skipped="0" failures="0" errors="0" timestamp="2017-11-01T03:52:32" hostname="Lenovo-PC" time="0.009">
  <properties/>
  <testcase name="convertTest" classname="com.linecorp.bot.client.ExceptionConverterTest" time="0.001"/>
  <testcase name="convertUnknownExceptionTest" classname="com.linecorp.bot.client.ExceptionConverterTest" time="0.001"/>
  <testcase name="exceptionInConvertFallbackedTest" classname="com.linecorp.bot.client.ExceptionConverterTest" time="0.007"/>
=======
<testsuite name="com.linecorp.bot.client.ExceptionConverterTest" tests="3" skipped="0" failures="0" errors="0" timestamp="2017-11-01T07:51:03" hostname="Ignaviers-MacBook-Pro.local" time="0.009">
  <properties/>
  <testcase name="convertTest" classname="com.linecorp.bot.client.ExceptionConverterTest" time="0.001"/>
  <testcase name="convertUnknownExceptionTest" classname="com.linecorp.bot.client.ExceptionConverterTest" time="0.0"/>
  <testcase name="exceptionInConvertFallbackedTest" classname="com.linecorp.bot.client.ExceptionConverterTest" time="0.008"/>
>>>>>>> 62e2cd5f49a66c63bf9f5eda34b412637cd1dcbb
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
